dspace-api/src/test/java/org/dspace/app/util/SubmissionConfigIT.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.dspace.AbstractIntegrationTestWithDatabase;
+import org.dspace.builder.CollectionBuilder;
+import org.dspace.builder.CommunityBuilder;
+import org.dspace.content.Collection;
+import org.dspace.content.Community;
+import org.dspace.submit.factory.SubmissionServiceFactory;
+import org.dspace.submit.service.SubmissionConfigService;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Integration Tests for parsing and utilities on submission config forms / readers
+ *
+ * @author Toni Prieto
+ */
+public class SubmissionConfigIT extends AbstractIntegrationTestWithDatabase {
+
+    SubmissionConfigService submissionConfigService;
+
+    @Before
+    @Override
+    public void setUp() throws Exception {
+        super.setUp();
+        submissionConfigService = SubmissionServiceFactory.getInstance().getSubmissionConfigService();
```

Comment:
I'd recommend deleting this entire `setUp()` method and simply calling  `SubmissionServiceFactory.getInstance().getSubmissionConfigService();` down in the `@Test` method below _when you need it._

I'm worried that calling this in the setUp may result in this getting called **before** the rest of the test database & other services initialize.

Reply:
Thanks for the suggestion, I've tried it but it hasn't worked either.